configure.ac

```diff
@@ -585,7 +585,9 @@ AC_CHECK_FUNCS(
         ftime \
         get_current_dir_name \
         getaddrinfo \
+	getdelim \
         getifaddrs \
+	getline \
```

Comment:
Why are these lines misaligned?
Is it a tabs and spaces thing?

Reply:
Yup; added a commit to retabify configure.ac's lists.